CONTRIBUTING.md

```diff
@@ -140,6 +140,18 @@ To start contributing to the project you would need to set up the project in you
   - `./scripts/check-all-the-things.sh` on Linux and MacOS
   - `.\scripts\check-all-the-things.ps1` on Windows
 
+- Integration tests are run automatically in Github Actions when a PR is locked, but it would be recommendable to run them locally before submitting a PR for review. You can find several scripts in `packages/framework-integration-tests/package.json` to run different test suites. You can run them using lerna too:
+
+- `lerna run integration`: Run all the integration test suites in the right order.
+- `integration/aws-deploy`: This test just checks that the sample project in `packages/framework-integration-tests/src` can be successfully deployed to AWS. The deployment process takes several minutes and this project is used by all the other AWS integration tests, so it's a requirement to run this test before.
+- `integration/aws-func`: AWS functional integration tests. They stress the deployed app write API and checks that the results are the expected ones both in the databases and the read APIs.
+- `integration/end-to-end`: Runs complete and realistic use cases on several cloud providers. This tests are intended to verify that a single project can be deployed to different cloud providers. Currently, only AWS is implemented though.
+- `integration/aws-nuke`: This test checks that the application deployed to AWS can be properly nuked. This test should be the last one after other test suites related to AWS have finished.
+- `integration/local`: Checks that the test application can be launched locally and that the APIs and the databases behave as expected.
+- `integration/cli`: Checks cli commands and check that they produce the expected results.
```

Comment:
Perhaps we should add it to each of those as well?

```suggestion
- `lerna run integration/aws-deploy`: This test just checks that the sample project in `packages/framework-integration-tests/src` can be successfully deployed to AWS. The deployment process takes several minutes and this project is used by all the other AWS integration tests, so it's a requirement to run this test before.
- `lerna run integration/aws-func`: AWS functional integration tests. They stress the deployed app write API and checks that the results are the expected ones both in the databases and the read APIs.
- `lerna run integration/end-to-end`: Runs complete and realistic use cases on several cloud providers. This tests are intended to verify that a single project can be deployed to different cloud providers. Currently, only AWS is implemented though.
- `lerna run integration/aws-nuke`: This test checks that the application deployed to AWS can be properly nuked. This test should be the last one after other test suites related to AWS have finished.
- `lerna run integration/local`: Checks that the test application can be launched locally and that the APIs and the databases behave as expected.
- `lerna run integration/cli`: Checks cli commands and check that they produce the expected results.
```

Reply:
I agree, and it would also be helpful to include the option `--stream`. If not you will wait for 10 minutes with the terminal blocked without seeing any logs (which probably will make you think something went wrong)

(see my suggestion so that we avoid repetition in the text)